mis_builder/models/mis_report_instance.py

```diff
@@ -944,6 +944,8 @@ def drilldown(self, arg):
                 account_id,
             )
             domain.extend(period._get_additional_move_line_filter())
+            context = arg.get("context") or {}
```

Comment:
This PR sounds interesting, but why looking for the context in `arg` instead of `self.env.context` ?

Reply:
Sorry for late reply. I get the context now from frozendict self.env.context.